index.html

```diff
@@ -122,6 +122,36 @@ <h1>Introduction</h1>
 </p>
 </section>
 
+<section>
+<h1>The Registration Process</h1>
+<p>
+Software implementers may find that the existing Decentralized
+Identifier Methods listed in this repository are not suitable for their
+use case and may need to add a new method to this registry.
+Adding a Decentralized Identifier Method to this list is designed to be a
+lightweight, community-driven process. In order to add a new method to this
+registry, an implementer MUST:
+</p>
+
+<ol>
+  <li>
+    Implement at least an experimental version of the new DID Method.
+  </li>
+  <li>Create a specification describing the new DID Method.</li>
+  <li>
+    Request that the specification is added to this registry by submitting a
+    Github Pull Request that adds the new suite to the list of existing suites.
```

Comment:
Why introduce the term "suite" here? I know it's used for cryptographic suites, but I think here we should just stick to the term "method" to avoid confusion?

Reply:
> Why introduce the term "suite" here?

Woops, copy/paste left over. I pulled the text from the cryptosuite registry. Will fix on next edit cycle.

Reply:
Fixed in https://github.com/w3c-ccg/did-method-registry/pull/5/commits/6608f7907c5b6832b8bb17f65d4063e436ef50ef.